### PR TITLE
adds completed exercise 7 (with minor amend to getArticleComments fr…

### PR DESCRIPTION
**Pull request by aBCodes23**
…om task 6

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -181,21 +181,91 @@ describe("/api/articles/:article_id/comments", () => {
...
+    .send(newComment)
     .expect(404)
-    .then(({ body }) => {
-      const { msg } = body;
-      expect(msg).toBe("Bad Request: Article does not exist");
-    });
+    .then(({body}) => {
+      const {msg} = body
+      expect(msg).toBe('Not Found: Article does not exist')
+    })
```
Comment (MRombi) [inline __tests__/app.test.js:285]: We should add a test for when the username is not valid 404 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -181,21 +181,91 @@ describe("/api/articles/:article_id/comments", () => {
...
     return request(app)
-    .get('/api/articles/article4/comments')
+    .post("/api/articles/1/comments")
+    .send(newComment)
+    .expect(201)
+    .then(({body}) => {
+      const {comment} = body
+      expect(comment).toMatchObject(desiredReturnedComment)
+    })
+  });
```
Comment (MRombi) [inline __tests__/app.test.js:228]: We should add a test for when there are unnecessary properties in the comment I.E. {fruit: 'apple', username: 'butter_bridge', body: 'this is a comment'}. We want to ignore the extra keys and send back 201 and the comment 🙂

**Review by MRombi — CHANGES_REQUESTED** [review]
Great job here, you are very close! Please add the tests I mentioned and resubmit 🙂 Let me know if you have any question 🙂

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the changes Alex, all good to merge!

**Pull request merged**